src/type/definition.js

```diff
@@ -568,6 +568,7 @@ function undefineIfEmpty<T>(arr: ?$ReadOnlyArray<T>): ?$ReadOnlyArray<T> {
 export class GraphQLScalarType {
   name: string;
   description: ?string;
+  specifiedByUrl: ?string;
```

Comment:
Just a note: this is technically a breaking change. If anyone was using `GraphQLScalarType` in their code, they now need to add `specifiedByUrl` in order to satisfy the type checker.